whisperx/asr.py

```diff
@@ -277,6 +278,7 @@ def load_model(whisper_arch,
         language: str - The language of the model. (use English for now)
         model: Optional[WhisperModel] - The WhisperModel instance to use.
         download_root: Optional[str] - The root directory to download the model to.
+        local_files_only: Optional[bool] - If `True`, avoid downloading the file and return the path to the local cached file if it exists.
```

Comment:
```suggestion
        local_files_only: bool - If `True`, avoid downloading the file and return the path to the local cached file if it exists.
```
Optional[...] is for arguments that default to None, which is not the case here.

Reply:
nvm, I didn't realise the PR was so old. I'll push the change